content/blogs/newsletter-103.md

```diff
@@ -0,0 +1,121 @@
+More specifically he fixed some issues in the Win32 subsystem kernel backend (`Win32k.sys`) where DirectX was not loaded hence the impossibility of accessing its APIs for use ([#4551](https://github.com/reactos/reactos/pull/4551)).
+
+This fixes an issue where it was not possible to change the display mode in VMWare as it attempts to invoke the DirectDraw API to handle the display management.
+Along with it, he worked on implementing DirectDraw management code in display mode switch ([#4519](https://github.com/reactos/reactos/pull/4519)).
+This improves the experience with games that are based upon DirectX stack.
+
+In the meantine Justin Miller is syncing up all of the DirectX DLLs with Wine up to the latest version ([#5688](https://github.com/reactos/reactos/pull/5688)).
+
+## NT6+ application compatibility preparations
+A lot of applications that we take for granted, be it games, browsers, E-mail clients, utilities and such rely on newer APIs provided by Windows.
+NT6+ is terminology that encompasses Windows operating systems like Vista, 7, 8, 8.1, 10 and 11 that base on a newer NT architecture design -- NT 6.0 starting with Windows Vista.
+At API compatibility level ReactOS currently aims for Windows Server 2003 (of which NT release version is 5.2) that is the last iteration in the NT5 family of Windows operating systems.
+
+As many software developers no longer support older versions of Windows, most of the applications that we use day to day will not be able to work on ReactOS. However this is going to change.
+A group made up by Timo Kreuzer, Justin Miller, and other developers and contributors alike are bringing the effort on developing the necessary APIs for compatibility with modern programs.
+While Timo is still working on implementing a dynamic versioning system for DLLs ([#3239](https://github.com/reactos/reactos/pull/3239)) that allows exporting of routines to applications depending on their compatibility settings, he has added the option to ReactOS bot builders to compile builds with NT6 exports which makes it possible to experiment with NT6+ application compatibility.
+
+{{< gallery >}}
+{{< figure link="/img/blogs/newsletter-103/obs-nt6.png" src="/img/blogs/newsletter-103/obs-nt6.png" caption="OBS 20.1 running on ReactOS (with some glitches)" >}}
+{{< figure link="/img/blogs/newsletter-103/obs-stream.png" src="/img/blogs/newsletter-103/obs-stream.png" caption="Streaming ReactOS running OBS 20.1" >}}
+{{< /gallery >}}
+
+Keep in mind that while NT6+ compatibility support is still experimental, this only focuses on user mode support.
+NT6+ support for kernel mode such as kernel drivers, the kernel itself as well other system core functionalities at low level are an entirely different subject.
+While we do aim to reach this goal as well in the future, it would require a lot of time and development effort therefore we currently cannot give an estimation as of when it will be done.
```

Comment:
```suggestion
While we do aim to reach this goal as well in the future, it would require a lot of time and development effort, therefore we currently cannot give an estimation as of when it will be done.
```

Reply:
After this last line I would add a short conclusion that connects with reader. Something like:

"Thank you for your interest in the ReactOS project. We look forward to sharing more announcements and progress updates with you.

Sincerely,

The ReactOS Team"